PodTemplates.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  labels:
+    jenkins: "agent"
+    job: "plugin-site"
+spec:
+  tolerations:
+    - key: "os"
```

Comment:
@olblak i removed highmem from the spec

do i need job and the os selector and stuff?

Reply:
seems like you have the OS one which looks good